### ISSUE TITLE
Two images do not have any accessible name, alt text, or title on main page

### ISSUE THREAD

**Issue by ollie-iterators** [CityOfBoston/Iterators#204]
Screengrab/videos (insert file names here)
2020 11 11, Images have no accessible name, alt, or title.mp4
[2020 11 11, Images have no accessible name, alt, or title.mp4.zip](https://github.com/CityOfBoston/Iterators/files/5524394/2020.11.11.Images.have.no.accessible.name.alt.or.title.mp4.zip)

Link
https://d8-stg.boston.gov/

Node # (if known)
21

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > MacOS 10.15.7 > Chrome 68

Trusted Tester Test
Images

Brief Description
Two images do not have any accessible name, alt text, or title on main page

Steps to reproduce
1)	00:00, Load the URL
2)	00:11, First image with no accessible name, alt, or title
3)	00:34, Second image with no accessible name, alt, or title

Desired Result
•	All images have accessible names, alt, or titles

Actual Result
•	There are two images on the page with no accessible names, alt, or titles

**Comment by ollie-iterators** [comment]
Links this also happens on:

- Node: 121, Link: https://d8-stg.boston.gov/departments/transportation. There are 4 images with no accessible name, alt, or title

**Comment by subaha-cob** [comment]
@jill-iterators I am not sure what I am looking for here. Are you talking about the SVG images? Those are background image replacements for when a news or event does not have an image. Please clarify your request. Thanks!

**Comment by subaha-cob** [comment]
@ollie-iterators The background images I see are purely decorative.

**Comment by ollie-iterators** [comment]
This appears to be fixed. I can't see the images that have the issue on the main page anymore.
